pyproject.toml

```diff
@@ -67,6 +67,21 @@ features = [
   "ci",
 ]
 
+[tool.ruff]
+# smallest version ruff seems to support
+line-length = 99
+target-version = "py37"
+include = ["pyproject.toml", "lib/spack/**/*.py", "var/spack/repos/**/*.py", "bin/spack"]
+exclude = ["lib/spack/external"]
```

Comment:
I would only ever use `extend-include` and `extend-exclude`, there are a lot of defaults we don't want to override such as excluding `.git`. No use including pyproject.toml, it can't be formatted anyway.